ipmi-sel/ipmi-sel-argp.c

```diff
@@ -96,9 +96,15 @@ static struct argp_option cmdline_options[] =
       "Display SEL records from record id START to END.", 44},
     { "exclude-display-range", EXCLUDE_DISPLAY_RANGE_KEY, "START-END", 0,
       "Exclude display of SEL records from record id START to END.", 45},
-    { "date-range", DATE_RANGE_KEY, "MM/DD/YYYY-MM/DD/YYYY", 0,
+    { "date-range", DATE_RANGE_KEY,
+      "MM/DD/YYYY-MM/DD/YYYY"
+      "\n                   "
+      "YYYY-MM-DDTHH:MM:SS-YYYY-MM-DDTHH:MM:SS", 0,
       "Display SEL records in the specified date range.", 46},
-    { "exclude-date-range", EXCLUDE_DATE_RANGE_KEY, "MM/DD/YYYY-MM/DD/YYYY", 0,
+    { "exclude-date-range", EXCLUDE_DATE_RANGE_KEY,
+      "MM/DD/YYYY-MM/DD/YYYY"
+      "\n                           "
+      "YYYY-MM-DDTHH:MM:SS-YYYY-MM-DDTHH:MM:SS", 0,
```

Comment:
this aligns for me in my terminal, and I assume it does in yours as well, but I don't know `argparse` well enough to know if this is safe in general.  Perhaps do "DATE1-DATE2".  And we can document the acceptable formats in the manpage instead (`man/ipmi-sel.8.pre.in`).  Perhaps can also say "See manpage for accepted date formats in the help output."

Reply:
I am definitely not more knowledgeable about argparse :) Thanks for the suggestions, I moved that info into the manpage in the update